src/vs/platform/backup/common/backup.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { createDecorator } from 'vs/platform/instantiation/common/instantiation';
+import Uri from 'vs/base/common/uri';
+
+export const IBackupService = createDecorator<IBackupService>('backupService');
+
+export interface IBackupService {
+	_serviceBrand: any;
+
+	getBackupWorkspaces(): string[];
```

Comment:
As discussed, I'll merge these services as there is no reason for them not to share code.